e2e/helpers/helpers.ts

```diff
@@ -92,8 +94,18 @@ export class HelpFunctions {
     await this.sessionAcceptButton.click();
   }
 
-  async assertCallActive() {
-    await expect(this.sessionStatus).toHaveText('active');
+  async assertCallStatus(status: string) {
+    await expect(this.sessionStatus).toHaveText(status);
+  }
+
+  async assertFirstSessionStatus(status: string) {
```

Comment:
Ah I see that you use status more often. Let's take it to the next level:

At the top of your file (under the imports) add this Typescript type:
```
type SessionStatus = 'ringing' | 'active' | 'on_hold';
```

Then you can change this function like so:
```suggestion
  async assertFirstSessionStatus(status: SessionStatus) {
```